### PR TITLE
Add libdxfrw/1.0.0

### PR DESCRIPTION
**Pull request by feragon**
Specify library name and version:  **libdxfrw/1.0.1**

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/wiki#how-to-submit-a-pull-request) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/libdxfrw/all/conanfile.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
+from conans import ConanFile, CMake, tools
+
+
+class LibdxfrwConan(ConanFile):
+    name = "libdxfrw"
+    license = "GPL2"
```
Comment (uilianries) [inline recipes/libdxfrw/all/conanfile.py:6]:
```suggestion
    license = "GPL-2.0-or-later"
```

File: recipes/libdxfrw/all/conanfile.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "C++ library to read/write DXF and read DWG files"
+    topics = ("dxf", "dwg")
```
Comment (uilianries) [inline recipes/libdxfrw/all/conanfile.py:9]:
```suggestion
    topics = ("dxf", "dwg", "cad")
```

File: recipes/libdxfrw/all/conandata.yml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+sources:
+  "1.0.1":
```
Comment (uilianries) [inline recipes/libdxfrw/all/conandata.yml:2]: The original project does not offer such version: https://sourceforge.net/projects/libdxfrw

Use the upstream version. You can this recipe as reference: https://github.com/conan-io/conan-center-index/blob/64deb844f5fbee9dd80325dd747d68e2a248c0ea/recipes/libconfuse/all/conanfile.py#L64

Comment (feragon) [reply recipes/libdxfrw/all/conandata.yml:2]: The original project on SourceForge is not maintained anymore : https://sourceforge.net/p/libdxfrw/discussion/general/thread/a71f71d3/#03ec/c469

This is the version we use in LibreCAD, which includes bug and security fixes.

File: recipes/libdxfrw/all/conanfile.py [hunk 4]
```diff
@@ -16,44 +20,43 @@ class LibdxfrwConan(ConanFile):
...
-                  excludes=('cmake', 'pkgconfig'))
-        self.copy("*", src="install/include", dst="include")
-        self.copy("COPYING", src=self._get_libdxfrw_folder(), dst="licenses")
+        tools.rmdir(os.path.join(self.package_folder, "lib", "cmake"))
+        tools.rmdir(os.path.join(self.package_folder, "lib", "pkgconfig"))
+        tools.rmdir(os.path.join(self.package_folder, "share"))
+
+    def package_info(self):
+        bin_path = os.path.join(self.package_folder, "bin")
+        self.output.info("Appending PATH env var: {}".format(bin_path))
```
Comment (uilianries) [inline recipes/libdxfrw/all/conanfile.py:58]: what binary is exported to the bin folder?

Comment (feragon) [reply recipes/libdxfrw/all/conanfile.py:58]: It's `dxf2dxf`
https://github.com/LibreCAD/libdxfrw/tree/master/dwg2dxf

Comment (uilianries) [reply recipes/libdxfrw/all/conanfile.py:58]: thanks!

**Pull request merged**